javascriptv3/example_code/cross-services/transcription-app/src/helper.ts

```diff
@@ -0,0 +1,83 @@
+/* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+ABOUT THIS NODE.JS EXAMPLE: This example works with AWS SDK for JavaScript version 3 (v3),
+which is available at https://github.com/aws/aws-sdk-js-v3. This example is in the 'AWS SDK for JavaScript v3 Developer Guide' at
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+Purpose:
+helper.ts is part of a tutorial demonstrating how to build and deploy an app that transcribes and displays
+voice recordings for authenticated users. To run the full tutorial, see
+https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+Running the code:
+For more information, see https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/transcribe-app.html.
+
+*/
+// Vanilla JavaScript helper functions for user interface
```

Comment:
Minor comment: code comments should typically be grammatically and syntactically complete sentences, including ending punctuation, especially for globalization.